syncopy/io/__init__.py

```diff
@@ -1,18 +1,21 @@
 # -*- coding: utf-8 -*-
-# 
+#
 # Populate namespace with io routines
-# 
+#
 
 # Import __all__ routines from local modules
-from . import utils, load_raw_binary, load_spy_container, save_spy_container
+from . import (utils, load_spy_container, save_spy_container,
+               read_external, _read_nwb)
 from .utils import *
-from .load_raw_binary import *
```

Comment:
This is legacy code that was kept around in case someday we might need it. It was intended to read a custom ESI binary format, however, it's so outdated now that modifying the existing code would probably take longer than writing a new routine from scratch